src/components/Pressable/utils.ts

```diff
@@ -119,12 +119,72 @@ const adaptTouchEvent = (event: GestureTouchEvent): PressableEvent => {
   };
 };
 
+type StylePropKeys = (keyof ViewStyle)[];
+
+// Source:
+// - From ViewStyle extracted FlexStyle sub-interface which contains all of the box-model manipulating props.
+// - From FlexStyle handpicked those styles, which act on the inner part of the box-model.
+const innerStyleKeys = [
```

Comment:
Then, if doing a single method to divide styles, this could be a set instead of an array for O(1) access time.

Reply:
Are you sure this is something we want?
We never access any members of this array besides iterating over the entire thing, 
at which point I don't think it can be optimised further than to `O(n)`, which I believe it currently is.

Reply:
See https://github.com/software-mansion/react-native-gesture-handler/pull/2982#discussion_r1668703763